circle.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
     - sed -i '/source \/home\/ubuntu\/virtualenvs\//d' ~/.circlerc
     - nvm alias default node
     - npm install -g csslint
+    - sudo apt-get install shellcheck
```

Comment:
This belongs in the other commit.

And I would expect that we also need to install shellcheck on Travis.  I am surprised it is already present on the travis VM.  I'll double check that in the next review.

p.s. I've created https://github.com/coala/coala-bears/issues/2317

Reply:
Done :sweat_smile:

Reply:
Also check https://travis-ci.org/coala/coala/jobs/350911877#L60.

Reply:
We dont need a third commit.  Adding shellcheck to Circle Ci is part of enabling shellcheck on Circle CI.